test/fluree/db/shacl/shacl_basic_test.clj

```diff
@@ -956,3 +960,73 @@
                                    "@type"                         "https://example.com/Actor",
                                    "https://example.com/name"      "Sam Worthington"})]
     (is (not (util/exception? db2)))))
+
+(deftest shacl-in-test
+  (testing "value nodes"
+    (let [conn   @(fluree/connect {:method :memory
+                                   :defaults
+                                   {:context test-utils/default-str-context}})
+          ledger @(fluree/create conn "shacl-in-test"
+                                 {:defaultContext ["" {"ex" "http://example.com/ns/"}]})
+          db0    (fluree/db ledger)
+          db1    @(fluree/stage db0 [{"type"           ["sh:NodeShape"]
+                                      "sh:targetClass" {"id" "ex:Pony"}
+                                      "sh:property"    [{"sh:path" {"id" "ex:color"}
+                                                         "sh:in"   '("cyan" "magenta")}]}])
```

Comment:
Does this list syntax desugar into `{"@list" ["cyan" "magenta"]}`?

Reply:
> Does this list syntax desugar into {"@list" ["cyan" "magenta"]}?

I don't think `sh:in` would need to be ordered, so it shouldn't matter if it is a `@list` or not, correct?

Reply:
Yeah it shouldn't matter if it's a list for the reason Brian states. But the spec says it is a list, so I just used that. There's no specific desugaring in place as of now since nothing is relying on @list-specific behavior yet.